presto-spi/src/main/java/io/prestosql/spi/eventlistener/EventListener.java

```diff
@@ -26,4 +26,8 @@ default void queryCompleted(QueryCompletedEvent queryCompletedEvent)
     default void splitCompleted(SplitCompletedEvent splitCompletedEvent)
     {
     }
+
+    default void tracerEventOccurred(TracerEvent tracerEvent)
```

Comment:
This method name sounds funny (I don't have any suggestions for alternate names, yet -- let me think about it).